modules/apps/frontend-theme/kyoung-liferay-theme/src/js/canvas.js

```diff
@@ -0,0 +1,89 @@
+var space;
+
+function floatySpace() {
+    var colors = [
+        '#FF3F8E', '#04C2C9', '#2E55C1'
+    ];
+
+    space = new CanvasSpace('canvas', '#252934').display();

```

Comment:
There should be a line break between JS variables unless they are similar values.
Ex.
```
var a = document.getElementById(...);
var b = document.getElementById(...);

var abc = document.querySelector(...);

var aaa = document.addEventListener(...);
```